azure-sfmesh-maven-plugin/pom.xml

```diff
@@ -0,0 +1,210 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+      <groupId>com.microsoft.azure</groupId>
+      <artifactId>azure-maven-plugins-pom</artifactId>
+      <version>1.0.0-SNAPSHOT</version>
+      <relativePath>../azure-maven-plugins-pom</relativePath>
+  </parent>
+
+  <groupId>com.microsoft.azure</groupId>
+  <artifactId>azure-sfmesh-maven-plugin</artifactId>
+  <version>0.1.0</version>
+  <packaging>maven-plugin</packaging>
+  <name>Maven Plugin for Service Fabric Mesh</name>
+  <description>Maven Plugin for Service Fabric Mesh Applications</description>
+  <url>https://github.com/microsoft/azure-maven-plugins</url>
+
+  <properties>
+    <azure.maven-plugin-lib.version>1.2.4-SNAPSHOT</azure.maven-plugin-lib.version>
```

Comment:
Please keep with the version of maven plugin lib https://github.com/Microsoft/azure-maven-plugins/blob/88f588fd156ded13a203103175967cbf440d09b1/azure-maven-plugin-lib/pom.xml#L16